qutip/bloch.py

```diff
@@ -38,16 +38,28 @@
 from numpy import (ndarray, array, linspace, pi, outer, cos, sin, ones, size,
                    sqrt, real, mod, append, ceil, arange)
 
+from packaging.version import parse as parse_version
+
 from qutip.qobj import Qobj
 from qutip.expect import expect
 from qutip.operators import sigmax, sigmay, sigmaz
 
 try:
+    import matplotlib
     import matplotlib.pyplot as plt
     from mpl_toolkits.mplot3d import Axes3D
     from matplotlib.patches import FancyArrowPatch
     from mpl_toolkits.mplot3d import proj3d
 
+    def matplotlib_version_gte(version='3.4'):
+        """
+        Checks if matplotlib version is greater than a specific version
+        """
+        if parse_version(matplotlib.__version__) >= parse_version(version):
+            return True
+        else:
+            return False
```

Comment:
`if x: return True; else: return False` is unnecessary - you can just `return x`, e.g.
```python
return parse_version(matplotlib.__version__) >= parse_version(version)
```

Reply:
Oh! I should have done that before! Thank you for the suggestion!